src/main/java/com/homeworks/homework20/interceptor/UserInterceptor.java

```diff
@@ -0,0 +1,35 @@
+package com.homeworks.homework20.interceptor;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+import org.springframework.web.servlet.ModelAndView;
+import org.springframework.web.servlet.handler.HandlerInterceptorAdapter;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@Component
+public class UserInterceptor extends HandlerInterceptorAdapter {
+    private static final Logger logger = LoggerFactory.getLogger(UserInterceptor.class);
```

Comment:
Naming is not right.
https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names